src/sentry/api/endpoints/issues/related_issues.py

```diff
@@ -9,22 +7,37 @@
 from sentry.api.bases.group import GroupEndpoint
 from sentry.issues.related import find_related_issues
 from sentry.models.group import Group
+from sentry.types.ratelimit import RateLimit, RateLimitCategory
 
 
 @region_silo_endpoint
 class RelatedIssuesEndpoint(GroupEndpoint):
     owner = ApiOwner.ISSUES
     publish_status = {"GET": ApiPublishStatus.EXPERIMENTAL}
+    enforce_rate_limit = True
+    rate_limits = {
+        "GET": {
+            RateLimitCategory.IP: RateLimit(15, 1),
```

Comment:
nit: I know that we don't do that elsewhere, but IMO it would be good practice to start writing these like this:

```suggestion
            RateLimitCategory.IP: RateLimit(limit=15, window=1),
```

Reply:
Works for me 👍🏻